waldo.py

```diff
@@ -0,0 +1,36 @@
+import cv2
+import random
+
+def find_waldo():
+    # Load the image
+    random_image = get_random_image()
+
+    image = cv2.imread(random_image)
+    
+    # Convert the image to grayscale
+    gray = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
+    
+    # Load the Waldo template image
+    template = cv2.imread(random_image, 0)
+    
+    # Display the image with the rectangle
+    cv2.imshow('Waldo', image)
+    cv2.waitKey(0)
+    cv2.destroyAllWindows(1)
+    
+
+def get_random_image():
+    image_list = ['A.jpg', 'B.jpg', 'C.jpg', 'D.jpg', 'E.jpg', 'F.jpg', 'G.jpg']

```

Comment:
Images missing. do a git push.
Please note: all such files like images, models are to be kept in media.